operations/power_management/Save_Management_Network_Switch_Configurations.md

```diff
@@ -4,70 +4,123 @@ Switches must be powered on and operating. This procedure is optional if switch
 
 **Optional Task:** Save management network switch configurations before removing power from cabinets or the CDU. Management switch names are listed in the `/etc/hosts` file.
 
-## Obtain the list of switches
-
-From the command line on any NCN run:
-
-```bash
-grep 'sw-' /etc/hosts
-```
-
-Example output:
-
-```
-10.252.0.2 sw-spine-001
-10.252.0.3 sw-spine-002
-10.252.0.4 sw-leaf-001
-ncn-m001:~ #
-```
-
 ## Save switch configs
 
 ### Aruba Switch and HPE Server Systems
 
-On Aruba-based systems all management network switches will be Aruba and the following procedure.
-For each switch:
-
-1. `ssh` to the switch
-1. Execute the `write memory` command
-1. Exit the switch shell
-
-Example:
-
- ```bash
- ssh admin@sw-spine-001.nmn
- admin@sw-spine-001 password:
- write memory
- exit
- ```
+On Aruba-based systems, all management network switches will be Aruba.
+
+1. (`ncn-m#`) Connect to all management network switches.
+
+   This loop will login to each switch as the admin username. Provide the password for that username. While logged in to each switch, some commands will be issued.  Then after typing `exit`, the loop will login to the next switch.
+
+    ```bash
+    for switch in $(awk '{print $2}' /etc/hosts | grep 'sw-'); do echo  "switch ${switch}:" ; ssh admin@$switch; done
+    ```
+
+   Commands to issue on each switch:
+
+   ```bash
+   write memory
+   exit
+   ```
+
+   Example output:
+
+   ```bash
+   switch sw-spine-001:
+   admin@sw-spine-001's password: 
+   ###############################################################################
+   # CSM version:  1.3
+   # CANU version: 1.6.20
+   ###############################################################################
+
+   Last login: 2024-01-29 00:58:44 from 100.113.129.22
+   User "admin" has logged in 2 times in the past 30 days
+   sw-spine-001# write memory
+   Copying configuration: [Success]
+   sw-spine-001# exit
+   Connection to sw-spine-001 closed.
+   switch sw-spine-002:
+   admin@sw-spine-002's password: 
+   ###############################################################################
+   # CSM version:  1.3
+   # CANU version: 1.6.20
+   ###############################################################################
+
+   Last login: 2024-01-29 01:01:44 from 100.113.129.22
+   User "admin" has logged in 1 times in the past 30 days
+   sw-spine-002# write memory
+   Copying configuration: [Success]
+   sw-spine-002# exit
+   Connection to sw-spine-002 closed.
+   etc.
```

Comment:
Elsewhere in this file you use `...` to truncate output. Suggest being consistent

```suggestion
   ...
```

Reply:
Fixed.